wikimedia/wikipedia-ios
# Update toolbar buttons in edit mode

**Phabricator:** 
https://phabricator.wikimedia.org/T301126

### Notes
* This PR updates the behavior of the buttons in `Edit Mode`. The  `Mark All As Read` button is now disabled when there are no unread notifications. The `Mark` button options now only display the Mark As Read option if the selection contains only unread notifications. Only the Mark As Unread option is shown when there are only read notifications. Both options are displayed when the selection contains both read and unread notifications.

### Test Steps
1. Go to the Notifications Center, tap the `Edit` button
2. Select some read notifications. Click the Mark button, it should only display the "Mark as unread" option.
3. Select some unread notifications. Click the Mark button, it should only display the "Mark as read" option.
4. Mark all notifications as unread (if not already), go back to Edit mode: the "Mark All As Read" button should be disabled.
5. Mark some notifications as unread, but not all. The "Mark All As Read" should be enabled and work normally.
